### ISSUE TITLE
enhancement: a11y rule "anchor-has-content" fails with a warning.

### ISSUE THREAD

**Issue by FatumaA** [open-sauced/app#4127]
### Describe the bug

[jsx-a11y eslint plugin](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) was recently added to the project and this rule - jsx-a11y/anchor-has-content - fails with a warning.

This issue offers more context: https://github.com/open-sauced/app/issues/2264

### Steps to reproduce

The following component and file is causing the rule to fail:

1. components/StarSearch/Chatbox.tsx

**Comment by michaella23** [comment]
.take

**Comment by FatumaA** [comment]
Hi, @michaella23 
No rush, but I wanted to ask how's this going?
